src/panels/config/network/supervisor-hostname.ts

```diff
@@ -51,22 +51,23 @@ export class HassioHostname extends LitElement {
       <ha-card
         class="no-padding"
         outlined
-        .header=${this.hass.localize("ui.panel.config.network.hostname.title")}
+        .header=${this.hass.localize(
+          "ui.panel.config.network.supervisor.hostname.title"
+        )}
       >
-        <div>
-          <ha-settings-row .narrow=${this.narrow}>
-            <span slot="heading">Hostname</span>
-            <span slot="description"
-              >The name your instance will have on your network</span
-            >
-            <ha-textfield
-              .disabled=${this._processing}
-              .value=${this._hostname}
-              @change=${this._handleChange}
-              placeholder="homeassistant"
-            >
-            </ha-textfield>
-          </ha-settings-row>
+        <div class="card-content">
+          <span slot="description"
```

Comment:
This slot will not do anything here I think?

Reply:
Removed